src/Logging.jl

```diff
@@ -77,7 +78,7 @@ const _root = Logger("root", WARNING, STDERR)
 Logger(name::AbstractString;args...) = configure(Logger(name, WARNING, [STDERR], _root); args...)
 Logger() = Logger("logger")
 
-write_log(syslog::SysLog, color::Symbol, msg::AbstractString) = send(syslog.socket, syslog.ip, syslog.port, msg)
+write_log(syslog::SysLog, color::Symbol, msg::AbstractString) = send(syslog.socket, syslog.ip, syslog.port, length(msg) > syslog.maxsize ? msg[1:syslog.maxsize] : msg)
```

Comment:
A bit confusing to call the variable `maxsize` and then check the string's `length()`. Consider using either `maxlength` as the name of the config var or `sizeof()` when checking to see if the message is too long.